src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
 import './App.css';
+import { Container } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Routes from './components/Routes'
+import Nav from './components/Navigation'
+import "./App.css";
+
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
```

Comment:
I don't think you need to wrap this in a `<div>` unless you plan on including additional elements below routes.
So, this simplifies to:

```
return <Routes />
```

Also, arrow function.

```
const App = () => {
}
```